applications/plugins/SofaCUDA/sofa/gpu/cuda/CudaMatrix.h

```diff
@@ -190,11 +190,11 @@ class CudaMatrix
         if (devicePointer!=NULL) mycudaFree(devicePointer);
     }
 
-    Size getSizeX() const {
+    Size colSize() const {
         return sizeX;
     }
 
-    Size getSizeY() const {
+    Size rowSize() const {
```

Comment:
In fact to ensure smooth transition for people using the old methods you could add:
```c++
[[deprecated("2021-xx-xx: Method has been depreciate in PR #1788. Please use rowSize instead"]]
Size getSizeY() const {return  rowSize();}
```

Reply:
It's a great idea, thanks @epernod !
I wasn't familiar with the deprecated decorator.
I'll update the branches accordingly before the dev meeting tomorrow